helms/legion/values.yaml

```diff
@@ -1,129 +1,459 @@
-rootDomain: k8s-nightly.epm.kharlamov.biz
-domainDelimiter: "-"
-namespace: default
-enclaveName: unknown
-localDomain: "cluster.local"
-addLocalDomain: false
-clusterName: ""
+# Version of Legion
+# By default .Chart.AppVersion version is used
+# Type: string
+# legionVersion: "1.0"
 
-log_level: info
+# Does cluster require RBAC
+# Type: bool
+rbac: true
 
-auth:
+# Docker images registry
+# This registry will be used for automatic docker image name deduction based on Legion naming for images
+# Each image could be overrided mannualy in .<service>.customImage, please see .edi root section or other
+# Type: string
+imagesRegistry: legionplatform
```

Comment:
Missed slash?
imagesRegistry: legionplatform/

Reply:
agree, I'll do it

Reply:
done